pcsx2/Pcsx2Config.cpp

```diff
@@ -234,6 +235,28 @@ void Pcsx2Config::GSOptions::LoadSave( IniInterface& ini )
 	IniEntry( FramesToSkip );
 }
 
+int Pcsx2Config::GSOptions::GetVsync() const
+{
+	// If we have a compositor (AKA not exclusive fullscreen), we need to
+	// wait the compositor signal. 0x10000 is a flag for the GS plugin
+	if (VsyncEnable == VSYNC_COMPOSITOR)
+		return 0x10000;
+
+	if (!FrameLimitEnable)
+		return 0;
+
+	if (g_LimiterMode == Limit_Turbo)
+		return 0;
```

Comment:
I wonder if these ``if()`` statements could be combined? (Avoids redundant return statement)